src/Psalm/Internal/Analyzer/Statements/Expression/Call/FunctionCallAnalyzer.php

```diff
@@ -789,7 +837,7 @@ private static function analyzeInvokeCall(
         $fake_method_call = new VirtualMethodCall(
             $function_name,
             new VirtualIdentifier('__invoke', $function_name->getAttributes()),
-            $stmt->getArgs()
+            $stmt->isFirstClassCallable() ? [new PhpParser\Node\VariadicPlaceholder()] : $stmt->getArgs()
```

Comment:
Please only create Virtual nodes from Psalm. VirtualVariadicPlaceholder may not exist yet, but it should be created

Reply:
I wonder if retrieving $stmt->args would be enough here, technically it already return the real VariadicPlaceholder when isFirstClassCallable is true

Reply:
Psalm was using $stmt->args everywhere a few weeks ago. I replaced them all when introducing support for PHP-Parser 4.13: https://github.com/vimeo/psalm/pull/6628